packages/css-library/dist/stylesheets/base/fonts.css

```diff
@@ -762,18 +762,25 @@ Converts a value in px to a value in rem
 */
 /*
 ----------------------------------------
-rem-to-px()
+rem-to-user-em()
 ----------------------------------------
-Converts a value in rem to a value in px
+Converts a value in rem to a value in
+[user-settings] em for use in media
+queries
 ----------------------------------------
 */
 /*
 ----------------------------------------
-rem-to-user-em()
+px-to-user-em()
```

Comment:
USWDS added a new Sass function in this update and comments for it are getting output in a bunch of stylesheets.